  File "C:\Users\F5505682\git\kongssdfnb\customer-service\app\main.py", line 3, in <module>
    from database import metadata, engine, DATABASE_URL
ImportError: cannot import name 'metadata' from 'database' (C:\Users\F5505682\git\kongssdfnb\customer-service\app\database.py)
